.travis.yml

```diff
@@ -28,10 +28,12 @@ matrix:
 # Define an environment variable
 env:
   - DB=mysql
+  - INI=~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
```

Comment:
this is nice :D

Reply:
@DavidBadura but not working correctly :-D still working on a running solution